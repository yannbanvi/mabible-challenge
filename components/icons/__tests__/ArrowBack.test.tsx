/** @jest-environment jsdom */
import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import ArrowBackIcon from '../ArrowBack'
describe('ArrowBackIcon', () => {
  it('should render ArrowBackIcon successfully', () => {
    const { container } = render(<ArrowBackIcon />)
    const svg = container.querySelector('svg');
    expect(svg).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  })
});