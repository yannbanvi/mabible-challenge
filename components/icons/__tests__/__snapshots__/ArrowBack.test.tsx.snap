// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowBackIcon should render ArrowBackIcon successfully 1`] = `
<div>
  <svg
    fill="none"
    height="6"
    viewBox="0 0 10 6"
    width="10"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M1 1L5 5L9 0.999999"
      stroke="#211B28"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</div>
`;
